test/unit/core/type_traits/template_inspection_test.cpp

```diff
@@ -93,27 +91,27 @@ struct bar2
 
 TEST(template_inspect, transfer_template_vargs_onto_enum)
 {
-    EXPECT_TRUE((std::is_same_v<detail::transformation_trait_or_t<
-                                    detail::transfer_template_vargs_onto<bar<e2::bar>, foo>,
+    EXPECT_TRUE((std::is_same_v<seqan3::detail::transformation_trait_or_t<
+                                    seqan3::detail::transfer_template_vargs_onto<bar<e2::bar>, foo>,
                                     void>,
                                 void>));
```

Comment:
another tough one :sob: 

maybe extract the line
`using foo_e2_bar = seqan3::detail::transfer_template_vargs_onto<bar<e2::bar>, foo>;`